hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/impl/TestHddsDispatcher.java

```diff
@@ -276,6 +283,13 @@ private HddsDispatcher createDispatcher(DatanodeDetails dd, UUID scmId,
     return hddsDispatcher;
   }
 
+  private void shutdownDispatcher(HddsDispatcher dispatcher){
+    if(dispatcher != null) {
+      dispatcher.shutdown();
+    }
+    ContainerMetrics.remove();
```

Comment:
I am not convinced whether this step to remove the ContainerMetrics belongs to a method called shutdownDispatcher, at least from the name it is not clear it will do so besides shutting down the dispatcher. Shouldn't we separate this two?

Reply:
actually dispatcher.shutdown() method do nothing, and I put it here to follow the principle of shutdown used objects. 
I will edit this part to just call ContainerMetrics.remove()